cockatrice/src/dlg_update.cpp

```diff
@@ -0,0 +1,114 @@
+#define HUMAN_DOWNLOAD_URL "https://bintray.com/cockatrice/Cockatrice/Cockatrice/_latestVersion"
+
+#include <QtNetwork>
```

Comment:
I'm fine if you even just expose the same signals that the network object exposes